rotkehlchen/tests/unit/accounting/test_default_settings.py

```diff
@@ -94,7 +94,7 @@ def fixture_accounting_pot(accountant, airdrops_taxable, gas_taxable, include_cr
 
 
 def _gain_one_ether(
-        transactions: 'TransactionsAccountant',
+        history_base_entries: 'HistoryBaseEntriesAccountant',
```

Comment:
Adjust names as explained before. here and everywhere else

```suggestion
        events_accountant: 'HistoryBaseEntriesAccountant',
```